package.json

```diff
@@ -57,13 +57,13 @@
     "eslint": "^1.7.3",
     "eslint-config-gulp": "^2.0.0",
     "expect": "^1.20.2",
-    "gulp": "gulpjs/gulp#4.0",
+    "gulp": "^4.0.0",
     "gulp-test-tools": "^0.6.1",
-    "jscs": "^2.3.5",
+    "jscs": "^3.0.7",
     "jscs-preset-gulp": "^1.0.0",
-    "marked-man": "^0.1.3",
+    "marked-man": "^0.2.1",
```

Comment:
What changed in this module?

Reply:
Just I think newer module is better. Doesn't `^0.1.3` represents the range `>=0.1.3 <0.2.0`?